domains/authorization/Authorization.API/API/Data/IRepository.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+
+namespace API.Data;
+
+public interface IRepository<T> where T : class
+{
+    T? Get(Guid id);
```

Comment:
We don't use Generic repositories in other solutions. Should we start introducing new pattern here?

Reply:
Probably not. Any subsystem you believe i should mimic, to stay consistent?

Reply:
We should stay consistent.

Reply:
Looking further into this. I could implement specific repositories for each entity, but is that truly necessary? Do we need to be able to fetch, or save, each entity individually?

Reply:
I found a middle way of doing it. Take a look :)